app/views/report/_document_list.html.erb

```diff
@@ -1,16 +1,49 @@
-<% content_for :head do %>
-  <script type="text/javascript">
-    report_model = {
-      total_rows: <%= @response['response']['numFound'] %>,
-      data_url: <%= report_data_url(params.to_unsafe_h.merge(format: 'json')).to_json.html_safe %>,
-      download_url: <%= report_download_url(params.to_unsafe_h).to_json.html_safe %>,
-      column_model: <%= Report::COLUMN_MODEL.to_json.html_safe %>
-    }
-  </script>
-<% end %>
+<div class="mt-3"
+     data-controller="object-reporter"
+     data-object-reporter-data-url-value=<%= report_data_url(params.to_unsafe_h.merge(format: 'json', rows: 99_999_999)) %>
```

Comment:
This was significantly easier than trying to page through Solr results dependent on how far down the user has scrolled.

So, we'll want to test reports with a large number of results to see how badly performance suffers. Cc: @andrewjbtw

Reply:
if performance is poor, i wonder if we can send a Solr query that only returns the potential fields of interest and avoids large fields of no interest.

Reply:
Yeah, that should certainly go on the list of changes to consider.